src/inverseRobot.js

```diff
@@ -23,8 +23,21 @@
  *
  * @return {object}
  */
+
 function inverseRobot(robot) {
   // write code here
+  const result = {};
```

Comment:
Checklist:
3.[NAMING] - use proper names for variables - if you want to name variable result, pick a better one :)